dev/environment

```diff
@@ -70,3 +70,6 @@ OIDC_AUDIENCE=pypi
 # Default to the reCAPTCHA testing keys from https://developers.google.com/recaptcha/docs/faq
 RECAPTCHA_SITE_KEY=6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI
 RECAPTCHA_SECRET_KEY=6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe
+
+# Example of Captcha backend configuration
+# CAPTCHA_BACKEND=warehouse.captcha.recaptcha.Service
```

Comment:
A value isn't needed for deployment, since the default is currently `recaptcha.Service`